demo/docbot/cluster_bootstrap.py

```diff
@@ -17,32 +17,32 @@
 
 
 
-def initialize_cluster_settings(client: OpenSearch) -> None:
+def initialize_cluster_settings(client: OpenSearch, cluster_setting=None) -> None:
```

Comment:
Here can we remove the template parameters? We really just need to pull out the parts that may be variable. For this one I think they can all stay.